test/unit/core/type_traits/function_test.cpp

```diff
@@ -7,7 +7,7 @@
 
 #include <gtest/gtest.h>
 
-#include <seqan3/core/type_traits/function.hpp>
+#include <seqan3/core/type_traits/basic.hpp>
```

Comment:
It's weird that the test called `function_test` is not including `seqan3/core/type_traits/function.hpp`
It currently just tests the `SEQAN3_IS_CONSTEXPR` macro? So maybe the test should be renamed?

Reply:
This is fixed in upcoming PRs. Otherwise I would have deleted this header to just reintroduce it in the next PR. That seemd kind of worthless to me

Reply:
ok